Tests/CMakeLists.txt

```diff
@@ -7,27 +7,31 @@ Licensed under Apache License v2.0 with Runtime Library Exception
 See https://swift.org/LICENSE.txt for license information
 #]]
 
-if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin) # FIXME Hook this up on Darwin
   find_package(dispatch CONFIG)
   find_package(Foundation CONFIG)
-endif()
-find_package(XCTest CONFIG)
+  find_package(XCTest CONFIG)
 
-add_executable(AtomicsTestBundle
-  AtomicsTests/main.swift
-  AtomicsTests/autogenerated/Basics.swift
-  AtomicsTests/DoubleWord.swift
-  AtomicsTests/LifetimeTracked.swift
-  AtomicsTests/LockFreeQueue.swift
-  AtomicsTests/LockFreeSingleConsumerStack.swift
-  AtomicsTests/StrongReferenceRace.swift
-  AtomicsTests/StrongReferenceShuffle.swift
-  AtomicsTests/UnsafeAtomicLazyReference.swift)
-target_link_libraries(AtomicsTestBundle PRIVATE
-  Atomics
-  XCTest
-  $<$<NOT:$<PLATFORM_ID:Darwin>>:dispatch>
-  $<$<NOT:$<PLATFORM_ID:Darwin>>:Foundation>)
+  add_executable(AtomicsTestBundle
+    AtomicsTests/main.swift
+    AtomicsTests/autogenerated/Basics.swift
+    AtomicsTests/DoubleWord.swift
+    AtomicsTests/LifetimeTracked.swift
+    AtomicsTests/LockFreeQueue.swift
+    AtomicsTests/LockFreeSingleConsumerStack.swift
+    AtomicsTests/StrongReferenceRace.swift
+    AtomicsTests/StrongReferenceShuffle.swift
+    AtomicsTests/UnsafeAtomicLazyReference.swift)
+  target_compile_definitions(AtomicsTestBundle PUBLIC
+    ENABLE_DOUBLEWIDE_ATOMICS)
 
-add_test(NAME SwiftAtomics
-  COMMAND AtomicsTestBundle)
+  target_link_libraries(AtomicsTestBundle PRIVATE
+    Atomics
+    _AtomicsShims
```

Comment:
Hmm, should the Shims be used by anyone using Atomics or is that for testing purposes?

Reply:
The shims module isn't just a header-only library -- it includes two (non-inlined) C functions for swift_retain_n/release_n.

Ideally these would be exported by the Atomics module, rather than forcing clients to link with the static shims lib. 

Or rather, ideally, the atomic strong references feature shouldn't inline calls to these into clients. 🤔

Reply:
> Or rather, ideally, the atomic strong references feature shouldn't inline calls to these into clients. 🤔

Ah, it looks like this may an easy fix -- the package currently defines the Swift wrappers for these as `@inlinable` functions, for no good reason. (I don't quite get why just having those would've caused build failures for me previously, but the failures seem to be gone now, so... ¯\_(ツ)_/¯)